app/deployment/layout.tsx

```diff
@@ -0,0 +1,121 @@
+'use client';
```

Comment:
Ideally, we want to have `use client` on small components where it's needed. Having it on the layout make pretty much the entire app `use client`.

Can we extract the part that need to be client side to its own component, and import this component here?